test/unit/controllers/ChoiceControllerSpec.scala

```diff
@@ -47,6 +47,7 @@ class ChoiceControllerSpec extends ControllerSpec with OptionValues with BeforeA
 
     authorizedUser()
     withCaching[Choice](Choice.choiceId, None)
+    withCaching(Choice.choiceId)
```

Comment:
why ?

Reply:
This is for `cache` method in our cache. So mock for caching data. In Movements our mocks are in separated methods